api-test-extension/src/tests/content_settings.js

```diff
@@ -66,9 +66,6 @@ var content_settings_test = new TestSet()
     .require("[Property Exists] cookies", methodExists(chrome.contentSettings, 'cookies'), { hideOnSuccess: true })
     .require("[Property Set] cookies", methodCall(chrome.contentSettings.cookies, 'set', detailsBlock, () => {}))
 
-    .require("[Property Exists] images", methodExists(chrome.contentSettings, 'images'), { hideOnSuccess: true })
-    .require("[Property Set] images", methodCall(chrome.contentSettings.images, 'set', detailsBlock, () => {}))
```

Comment:
It crashes the debug version of browser because it is not supposed to be accessed on Android. Extension APIs are just facade that exposes inner features. If inner features are covered with debug time assertions it is completely non-sense to remove them just to make this code to work.